project/ScalafixBuild.scala

```diff
@@ -342,4 +342,16 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
       }
     }
   )
+  private def testRulesFor(scalaVersion: String, state: State): State = {
+    supportedScalaVersionforRules(scalaVersion)
+      .filter(_ != scalaVersion)
+      .flatMap { v =>
+        List(
+          s"""set testsInput/scalaVersion := "$v"""",
```

Comment:
looking at the logs, it's not clear what version is set 
> 2020-07-01T13:07:50.4524207Z [info] Defining testsInput / scalaVersion
2020-07-01T13:07:50.4525011Z [info] The new value will be used by testsInput / Compile / compileBinaryFileInputs, testsInput / Compile / compileSourceFileInputs and 34 others.
2020-07-01T13:07:50.4525764Z [info] 	Run `last` for details.

so maybe we can make it explicit for troubleshooting?

```suggestion
          s"""set testsInput/scalaVersion := "$v"""",
          "show testsInput/scalaVersion",
```